### PR TITLE
Code cleanup and project restructuring

### PR DESCRIPTION
**Pull request by Ismael034**
Hi,

Sorry for creating a new PR. Messed up with branches. I have done some restructuring of the project. Having everything in just one package was confusing for me. I also have done some code cleanup. 

Thanks!

### CODE REVIEW

File: ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/constant/Constants.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package de.szalkowski.activitylauncher.constant;
+
+public class Constants {
+    public static final String PREF_HIDE_PRIVATE = "hide_private";
```
Comment (butzist) [inline ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/constant/Constants.java:4]: All these constants are related to preferences. Move to preferences package.

**Review by butzist — CHANGES_REQUESTED** [review]
Thank you for the MR - when I googled for Android project folder best practices, I found the following article: https://medium.com/@andycherkashyn/how-to-organize-android-project-files-the-right-way-46b34289ad0a

They suggest to group files by activity. Also following an MVC/MVP schema, they put the data models in a separate top-level folder. For us, this would be all the files in object, manager, util, and thirdparty.

File: ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/constant/Constants.java (hunk 1 above)
Comment (Ismael034) [reply ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/constant/Constants.java:4]: They are related yes, but also used in different packages of the app, like the base activity or the IconLauncher class.

**Comment by Ismael034** [comment]
There are a lot of different ways to organize a project. In this case, I think we should have a separate folder for "utils" in the root of the project as the classes in there are not data models

**Comment by butzist** [comment]
In my experience "utils" folders always end up accumulating unrelated functionality. Other than the signing functions, most of them are abstractions around Android APIs like preferences and package manager. Can we try to put them into a subpackage of the corresponding activity? Ideas for names: actions, system, bindings, wrappers, os

**Review by gufalex13 — APPROVED** [review]

**Pull request closed without merging**